### PR TITLE
Interfaces

### PR DESCRIPTION
**Pull request by cerilacanda**

### CODE REVIEW

File: content/Interfaces/Interfaces.q.md [hunk 1]
```diff
@@ -0,0 +1,2450 @@
...
+
+Which statements correctly describe the error?
+
+ - On line 2 of `LifeCycle.php`, the declaration `interfaces LifeCycle` is invalid.
+
+ - There is no semicolon `;` at the end of the statement on line 2 of `LifeCycle.php`.
+
+ - The `interface` keyword is misspelled as `interfaces` on line 2 of `LifeCycle.php`.
+
+ - There is no open curly `{` after `interfaces LifeCycle` on line 2 of `LifeCycle.php`.
```
Comment (redgeneLauren) [inline content/Interfaces/Interfaces.q.md]: Suggestion on: `open curly {`
What if: `open curly brace {`

File: content/Interfaces/Interfaces.q.md [hunk 2]
```diff
@@ -0,0 +1,2450 @@
...
+
+Given the `Animal` class and `LifeCycle` interface, write a program that uses an `implements` keyword to implement an interface. In the `Mammal.php` tab, use a `class` keyword to declare a class named `Mammal` that `extends` the `Animal` class and `implements` the `LifeCycle` interface. Within the curly braces `{}`, add an implementation or method body to all abstract methods of the `LifeCycle` interface. Inside the `stage()` method body, add a statement `return "adult";`. Inside the `species()` method body, add a statement that returns the string `animal`. Then, add another `public` method `display()` that overrides the abstract `display()` method of the `Animal` class. Inside the `display()` method body, add an `echo` statement to display the string `"The " . parent::getType() . " is an " . $this->stage() . " " . $this->species() . "."`. After the class declaration, add a statement that creates the `$petMammal` object an instance of the `Mammal` class passing the arguments `Cat` and `3`. Then, add another statement that calls the `display()` method of the `$petMammal` object. Run the program to view the output.
```
Comment (redgeneLauren) [inline content/Interfaces/Interfaces.q.md]: What if: ...write a program that uses the `implements` keyword to implement an interface.
To reduce redundancy of the word `an`

Comment (redgeneLauren) [inline content/Interfaces/Interfaces.q.md]: What if: In the `Mammal.php` tab, use the `class` keyword to declare a class named `Mammal`.....

Comment (redgeneLauren) [inline content/Interfaces/Interfaces.q.md]: What if: Inside the `stage()` method body, add the statement `return "adult";`.

File: content/Interfaces/Interfaces.facts.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+{
+    public function sayHello();
+}
+```
+
+The `implements` keyword lets a class implement an interface.
+
+A class can implement several interfaces. 
+
+A class that implements an interface is required to provide a method body or implementation to all the abstract methods specified in an interface.
```
Comment (redgeneLauren) [inline content/Interfaces/Interfaces.facts.md:24]: What if: 
A class that implements an interface requires to provide a method body or implementation to all the abstract methods specified in an interface.

File: content/Interfaces/Interfaces.q.md (hunk 2 above)
Comment (cerilacanda) [reply content/Interfaces/Interfaces.q.md]: This is already discussed before that we use the article a in part 4 and part 2. Hope you can still recall. Please check. Thanks.

File: content/Interfaces/Interfaces.facts.md (hunk 3 above)
Comment (cerilacanda) [reply content/Interfaces/Interfaces.facts.md:24]: I would prefer to use the original statement for clarity but if this is something wrong with the grammar then I can rephrase this. Is there something wrong with the original statement? Please check. Thanks.

File: content/Interfaces/Interfaces.q.md (hunk 2 above)
Comment (cerilacanda) [reply content/Interfaces/Interfaces.q.md]: We use the article `a` on our previous statements in part 4 in all our lesson like `Write a program that uses a for loop...` and etc. This has been discussed before. Please check. Thanks.

File: content/Interfaces/Interfaces.q.md (hunk 1 above)
Comment (cerilacanda) [reply content/Interfaces/Interfaces.q.md]: My mistake, thank you.

**Review by redgeneLauren — APPROVED** [review]

**Review by rosoarv — APPROVED** [review]

**Review by gjoren4 — APPROVED** [review]
Looks Good!

**Pull request merged**